mmdet3d/datasets/nuscenes_dataset.py

```diff
@@ -99,7 +99,7 @@ def _filter_with_mask(self, ann_info: dict) -> dict:
         """
         filtered_annotations = {}
         if self.use_valid_flag:
-            filter_mask = ann_info['bbox_3d_isvalid']
+            filter_mask = ann_info['bbox_3d_isvalid'] > 0
```

Comment:
`bbox_3d_isvalid` seems store the bool value?

Reply:
v1.0-mini of nusences is float

Reply:
I just test the v1.0-mini of nusence limited by my device, the bbox_3d_isvalid type is float

Reply:
A little strange here, we need to check it again

Reply:
I trained `nuscenes dataset` both for lidar detection and monocular detection, and everything seems to be OK. Do you use the latest code to update infos and train model?

Reply:
I got the error when I call ./tools/create_data.py with version=v1.0-mini rather than doing detection.

Reply:
I re-generate the nuscenes info with  `v1.0-mini` version just now, and everything is OK.

Reply:
maybe I need to upgrade my raw dataset, thanks very much

Reply:
I will download newest version and try it again